### PR TITLE
Add `not_applicable` and `inactive` counts + deprecate fields on breakdown response

### PR DESCRIPTION
**Pull request by GMadorell**
[sc-9935]
[sc-10269]

### CODE REVIEW

File: com/coralogix/xdr/v1/test/security_test_breakdown.proto [hunk 1]
```diff
@@ -19,6 +19,8 @@ message CountByResult {
   google.protobuf.UInt32Value passed = 1;
   google.protobuf.UInt32Value failed = 2;
   optional SeveritiesCount failed_severities = 3;
+  google.protobuf.UInt32Value inactive = 4;
+  google.protobuf.UInt32Value unrelated = 5;
```
Comment (GMadorell) [inline com/coralogix/xdr/v1/test/security_test_breakdown.proto:23]: other names I thought of:
- unknown
- not_executed

File: com/coralogix/xdr/v1/test/security_test_breakdown.proto [hunk 2]
```diff
@@ -29,11 +31,9 @@ message SecurityTestItem {
...
 
 message SecurityTestBreakdown {
-  reserved 5;
-  reserved "count_by_severity";
+  reserved 5, 2, 4;
+  reserved "count_by_severity", "security_test_items", "all_count";
   google.protobuf.StringValue execution_id = 1; // UUID
-  repeated SecurityTestItem security_test_items = 2;
   google.protobuf.Timestamp executed_at = 3;
-  google.protobuf.UInt32Value all_count = 4;
```
Comment (GMadorell) [inline com/coralogix/xdr/v1/test/security_test_breakdown.proto:37]: I searched in frontend I couldn't find any usages.

File: com/coralogix/xdr/v1/test/security_test_breakdown.proto [hunk 3]
```diff
@@ -29,11 +31,9 @@ message SecurityTestItem {
 }
 
 message SecurityTestBreakdown {
-  reserved 5;
-  reserved "count_by_severity";
+  reserved 5, 2, 4;
+  reserved "count_by_severity", "security_test_items", "all_count";
   google.protobuf.StringValue execution_id = 1; // UUID
-  repeated SecurityTestItem security_test_items = 2;
```
Comment (GMadorell) [inline com/coralogix/xdr/v1/test/security_test_breakdown.proto:35]: There are some usages right now in frontend, in a deprecated page which I'm gonna remove in a webapp PR before merging this PR.

File: com/coralogix/xdr/v1/test/security_test_breakdown.proto (hunk 1 above)
Comment (xBATx) [reply com/coralogix/xdr/v1/test/security_test_breakdown.proto:23]: Or maybe `inactive` -> `deactivated` if we would like to be consistent but I think it's fine either way.

Regarding `unrelated` - is filter going to be applied somehow on top of that? Or only `securityRuleFilter`?

Comment (xBATx) [reply com/coralogix/xdr/v1/test/security_test_breakdown.proto:23]: Regarding `unrelated` - what do you think about `not_applicable`? I think it's fine either way too :)

Comment (stsatlantis) [reply com/coralogix/xdr/v1/test/security_test_breakdown.proto:23]: `not_applicable` for the win :)

Comment (GMadorell) [reply com/coralogix/xdr/v1/test/security_test_breakdown.proto:23]: let's go with `not_applicable` then

**Review by xBATx — APPROVED** [review]

**Review by stsatlantis — APPROVED** [review]

**Pull request merged**